lib/conjur/policy/yaml/handler.rb

```diff
@@ -229,6 +229,11 @@ def alias anchor
   
           # Begins a new map entry.
           def scalar value, tag, quoted, anchor
+            if @member_count.key?(value)
+              raise "Duplicate attribute of type #{value}"
+            else
+              @member_count[value] = 1;
+            end
```

Comment:
i see that `member_count` is used as a boolean (key exists in the hash) so setting it to 1 is redundant. Did you consider using another data structure for this? If you're only using it to check if it exists in the data structure, why use a key-value one?

Reply:
fixed